api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirTaskDao.java

```diff
@@ -19,14 +19,15 @@
 import org.hl7.fhir.r4.model.DomainResource;
 import org.openmrs.module.fhir2.FhirTask;
 
-public interface FhirTaskDao {
+public interface FhirTaskDao extends FhirDao<FhirTask> {
 	
-	FhirTask saveTask(FhirTask task);
-	
-	FhirTask getTaskByUuid(@NotNull String taskUUID);
+	FhirTask getTask(@NotNull String taskUUID);
 	
+	//Do we need this line
```

Comment:
I'd remove it. And I'd remove the corresponding parts from the `FhirTaskService` class, since it can be handled by the `searchForTasks()` method.